issuer_controller/src/app.py

```diff
@@ -1,35 +1,30 @@
 #!/usr/bin/env python
 from flask import Flask, jsonify, abort, request, make_response
-from flask_script import Manager, Server
 
 import requests
 import json
 import os
 import time
 import yaml
-
 import config
-
 import issuer
 
-
 # Load application settings (environment)
 config_root = os.environ.get('CONFIG_ROOT', '../config')
 ENV = config.load_settings(config_root=config_root)
 
-# custom server class to inject startup initialization routing
-class CustomServer(Server):
-    def __call__(self, app, *args, **kwargs):
+class BCRegController(Flask):
+    def __init__(self):
+        print("Initializing " + __name__ + " ...")
+        super().__init__(__name__)
         issuer.startup_init(ENV)
-        #Hint: Here you could manipulate app
-        return Server.__call__(self, app, *args, **kwargs)
-
-app = Flask(__name__)
-manager = Manager(app)
 
-# Remeber to add the command to your Manager instance
-manager.add_command('runserver', CustomServer())
+app = BCRegController()
```

Comment:
Nitpick: we may want to rename this to just `Controller` for the issuer-controller project.